_overviews/scala3-book/interacting-with-java.md

```diff
@@ -192,7 +191,7 @@ Here are a few things to notice in that code:
 
 ## How to use Scala `Option` in Java
 
-When you need to use a Scala `Option` in your Java code, you can convert the `Option` to a Java `Optional` value using the `toJava` method of the Scala _scala.jdk.javaapi.OptionConverters_ object.
```

Comment:
do we now use ticks everywhere consistently? I know @alvinj was using underscore before, because O'Reilly did that.

Reply:
Ah, I tried to catch that as much as I could, but I might have missed some. I think we should use backticks everywhere for fully qualified names.